### PR TITLE
Weekly coordination meeting, 2019-12-05

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2019/2019-12-05-coordination.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+title: "HSF Weekly Meeting #175, 5 December, 2019"
+layout: meetings
+---
+
+# {{page.title}}
+
+Present/Contributors: Caterina Doglioni, Graeme Stewart, David Lange, Eduardo Rodrigues, Chris Jones, Marco Clemencic, Daniel Elvira, Pere Mato, Benedikt Hegner, Witek Pokorski, Andrea Valassi, Heather Gray, Michel Jouvin, Ben Morgan, Liz Sexton-Kennedy, Gloria Corti, Kyle Knoepfel, Stefan Roiser, Marco Clemencic, Sudhir Malik, Adam Parker, James Walder, David Lange, Caterina Doglioni
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-12-05-coordination.md]: Caterina is written down twice ;-)

File: organization/_posts/2019/2019-12-05-coordination.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    - You can pre-book already the first 10-ish days of July!
+
+### Training
+- Successful Software Carpentries at CERN 27-29 November 2019. We had up to 60 students. It covered basic unix skills, python, git, and several HEP+python tools (PyROOT, uproot, SWAN, etc.)
+    - Note places filled very quickly - there is a lot of demand!
+- Here is the [agenda](https://indico.cern.ch/event/834411/) and some [photos](https://indico.cern.ch/event/834411/page/19193-photos)
+- We are still collecting feedback from students - but initial survey results were quite positive
+- The organizers discussed possibly doing this again in the spring as there appears to be plenty of demand. March 25-27 would happen to work well for both CMS (week before CMS week) and ATLAS (week before training courses at CERN that would build on this one). Plenty of ideas for how to change/improve to be discussed in the meanwhile...
+
+- Analysis Preservation Bootcamp in February (to follow up with Sam M). ...Samuel said that it is oversubscribed 51 participants for 30 places. So definitley in demand and well advertised..due to demand a second one is being planned in April..
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-12-05-coordination.md]: Typo for "definitely". Punctuation is not OK in a couple of places.

File: organization/_posts/2019/2019-12-05-coordination.md (hunk 1 above)
Comment (graeme-a-stewart) [reply organization/_posts/2019/2019-12-05-coordination.md]: They get so much done it's almost deserved!

Comment (eduardo-rodrigues) [reply organization/_posts/2019/2019-12-05-coordination.md]: :-)!

**Comment by graeme-a-stewart** [comment]
Thanks @eduardo-rodrigues - tidied up those points. I'll leave it a little while before merging in case other people want to take a look.

File: organization/_posts/2019/2019-12-05-coordination.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+### Training
+- Successful Software Carpentries at CERN 27-29 November 2019. We had up to 60 students. It covered basic unix skills, python, git, and several HEP+python tools (PyROOT, uproot, SWAN, etc.)
+    - Note places filled very quickly - there is a lot of demand!
+- Here is the [agenda](https://indico.cern.ch/event/834411/) and some [photos](https://indico.cern.ch/event/834411/page/19193-photos)
+- We are still collecting feedback from students - but initial survey results were quite positive
+- The organizers discussed possibly doing this again in the spring as there appears to be plenty of demand. March 25-27 would happen to work well for both CMS (week before CMS week) and ATLAS (week before training courses at CERN that would build on this one). Plenty of ideas for how to change/improve to be discussed in the meanwhile...
+
+- Analysis Preservation Bootcamp is organised for 17-19 February
+    - Samuel said that it is oversubscribed 51 participants for 30 places, aso definitley in demand and well advertised
```
Comment (eduardo-rodrigues) [inline organization/_posts/2019/2019-12-05-coordination.md]: Sorry: aso definitely -> also definitely

**Pull request merged**